engine/runtime/src/main/java/org/enso/interpreter/node/callable/resolver/HostMethodCallNode.java

```diff
@@ -80,8 +106,15 @@ public boolean isInteropLibrary() {
    */
   public static PolyglotCallType getPolyglotCallType(
       Object self, String methodName, InteropLibrary library) {
-    if (library.isDate(self) && !library.isTime(self)) {
-      return PolyglotCallType.CONVERT_TO_DATE;
+    if (library.isDate(self)) {
+      if (library.isTime(self))
```

Comment:
Nitpick: I'd use `{` braces.

Reply:
Sure. Other nitpick could be that this pattern of `isDate`, `isTime`, `isTimezone` is repeated all over the place. Not a fan of this solution but at least there aren't more cases.